modules/extensions-mule-support/pom.xml

```diff
@@ -15,9 +15,13 @@
     </description>
 
     <properties>
-        <surefire.args>
-            ${surefire.test.unit.open.args}
-        </surefire.args>
+        <surefire.module.path>
+            ${org.mule.runtime.boot:mule-module-jpms-utils:jar}
+        </surefire.module.path>
+        <surefire.add.modules>
+            org.mule.runtime.jpms.utils
+        </surefire.add.modules>
```

Comment:
Is it ok that these are overriding the ones defined in the parent instead of appending?

Reply:
i just left the ones in the parent as a valid default, but the idea is to override them as needed, yes